src/app/issuer/components/issuer-list/issuer-list.component.ts

```diff
@@ -136,6 +181,65 @@ export class IssuerListComponent extends BaseAuthenticatedRoutableComponent impl
 				});
 			}
 		});
+
+		this.userProfileApiService.getIssuerStaffRequests().then((r) => (this.staffRequests = r.body));
+	}
+
+	async issuerSearchChange() {
+		console.log('search changed');
+		if (this.issuerSearchQuery.length >= 3) {
+			this.issuersLoading = true;
+			try {
+				this.issuerSearchResults = [];
+				this.issuerSearchResults = await this.publicApiService.searchIssuers(this.issuerSearchQuery);
+				console.log(this.issuerSearchResults[0].image);
+			} catch (error) {
+				this.messageService.reportAndThrowError(`Failed to issuers: ${error.message}`, error);
+			}
+			this.issuersLoading = false;
+			this.issuerSearchLoaded = true;
+		}
+	}
+
+	selectIssuerFromDropdown(issuer) {
+		this.issuerSearchQuery = issuer.name;
+		this.selectedIssuer = issuer;
+		console.log(this.selectedIssuer);
+	}
+
+	ngAfterViewInit() {
+		this.issuerSearchInputModel.valueChanges
+			.pipe(debounceTime(500))
+			.pipe(distinctUntilChanged())
+			.subscribe(() => {
+				this.issuerSearchChange();
+			});
+	}
+
+	closeDialog() {
+		if (this.dialogRef) {
+			this.dialogRef.close();
+		}
+	}
+
+	requestMembership() {
+		this.issuerStaffRequestApiService.requestIssuerStaffMembership(this.selectedIssuer.slug).then(
+			(res) => {
+				if (res.ok) {
+					this.closeDialog();
+					this.staffRequests.push(res.body as ApiStaffRequest);
+					this.openSuccessfullyRequestedMembershipDialog();
+				}
+			},
+			(error) => {
+				this.closeDialog();
+				const err = BadgrApiFailure.from(error);
+				BadgrApiFailure.messageIfThrottableError(err.overallMessage) ||
+					''.concat(this.translate.instant('Issuer.addMember_failed'), ': ', err.firstMessage);
+				//@ts-ignore
+				this.messageService.reportAndThrowError(err.payload.response.error.detail);
```

Comment:
Just want to learn why  it would complain about this line, making `@ts-ignore` necessary